__tests__/integration.test.js

```diff
@@ -226,6 +226,32 @@ describe("Endpoint integration", () => {
 				});
 		});
 	});
+	describe("10. POST /api/articles/:article_id/comments", () => {
+		test(`should post a comment to specified article_id,
+		the body should accept and object with following properties:
+		username
+		body.`, () => {
+			const newComment = {
+				username: "butter_bridge",
+				body: "There's no shame in fear, my father told me, what matters is how we face it.",
+			};
+			return request(app)
+				.post("/api/articles/1/comments")
+				.send(newComment)
+				.expect(201)
+				.then(({ body }) => {
+					const { insertedComment } = body;
+					expect(insertedComment).toEqual({
+						article_id: 1,
+						comment_id: expect.any(Number),
+						votes: expect.any(Number),
+						created_at: expect.any(String),
+						author: "butter_bridge",
+						body: "There's no shame in fear, my father told me, what matters is how we face it.",
+					});
+				});
+		});
+	});
```

Comment:
Need to see some tests for the following error cases before the next pull request:
- [ ] Status 201, created comment object
  - [ ] Status 400, invalid ID, e.g. string of "not-an-id"
  - [ ] Status 404, non existent ID, e.g. 0 or 9999
  - [ ] Status 400, missing required field(s), e.g. no username or body properties
  - [ ] Status 404, username does not exist
  - [ ] Status 201, ignores unnecessary properties